src/components/data/viewers/ids.js

```diff
@@ -8,16 +8,19 @@ export default {
     DELETE_MENU_ITEM: "deleteRowMenuItem",
     DOWNLOAD_BTN: "downloadButton",
     DOWNLOAD_MENU_ITEM: "downloadMenuItem",
+    FILE_TYPE_DIALOG:"fileTypeDlg",
+    FILE_TYPE_SELECT:"fileTypeSelect",
```

Comment:
Looks like the GitHub action for checking the format failed - I see at least here there's no space after the `:`